src/apache-subversion.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+from bs4 import BeautifulSoup
+from common import dates, http, releasedata
+
+# https://regex101.com/r/CwvyT3/2 only non beta versions
+VERSION_PATTERN = re.compile(r"^(?:Subversion\s)(\d.\d+\.\d+$)")
```

Comment:
```suggestion
VERSION_PATTERN = re.compile(r"^Subversion\s(\d.\d+\.\d+$)")
```

Remove unnecessary non-capturing group.